### ISSUE TITLE
Support for YOLO v3

### ISSUE THREAD

**Issue by b-straub** [apple/turicreate#650]
Any plans to support YOLO v3 as object detector backend?
[YOLO v3](https://pjreddie.com/darknet/yolo/)

**Comment by gustavla** [comment]
Thanks for the feature request! Actually, even though our object detector pre-dates v3, we departed from v2 in a few ways that makes it more similar to v3. For instance, we use a softmax cross entropy loss for classes, as opposed to the regression loss in v2. Of course, there are still many differences and our model is still closer to v2.

It would be great to know in more detail why you are interested in v3. Is it purely for the impressive numbers, or are there particular aspects of v3 that you are interested in?

**Comment by b-straub** [comment]
Im interested in a higher accuracy for small objects. Watching this video
[YOLO v2 vs YOLO v3 vs Mask RCNN vs Deeplab Xception](https://www.youtube.com/watch?v=s8Ui_kV9dhw) I get the feeling that YOLO v3 is indeed doing a better job.

**Comment by gustavla** [comment]
Great, that is really helpful to know! There are indeed efforts in v3 to tackle this that I agree we could benefit from as well (using their three tiers of scale).

**Comment by pcmanik** [comment]
@gustavla And what about #140 ?

**Comment by gustavla** [comment]
@pcmanik Thanks for bringing your unanswered question to my attention. I answered on that issue and feel free to respond with feedback there.

**Comment by willbattel** [comment]
Has there been any update on this recently, or still just a pipeline dream? 🙃

**Comment by TobyRoseman** [comment]
related to #1734
